pyproject.toml

```diff
@@ -32,19 +32,19 @@ classifiers = [
 
 [tool.poetry.dependencies]
 
-python = ">=3.8.0,<3.12"
+python = ">=3.8.0,<3.13"
 
 # urllib must be pinned to <2 due to an issue with the `requests_toolbelt`
 # dependency in `gql`.
 # Similar: https://github.com/apache/arrow/commit/4295e3dd06f9f507f75ccc780eaca16a6b621a02
 urllib3 = "<2"
```

Comment:
could we unpin this? we have a couple dependencies that require `urllib3>=2` that we'd have to downgrade. I believe I was able to remove the dependency in https://github.com/anvilco/python-anvil/pull/65 and things still worked fine, but I don't have all the context